provider/rancher/rancher.go

```diff
@@ -517,17 +554,32 @@ func (lbp *LBProvider) createRancherLBService(lbConfig *config.LoadBalancerConfi
 	}
 	imageUUID = fmt.Sprintf("docker:%s", imageUUID)
 
+	scale := 0
+	if scaleStr, ok := lbConfig.Annotations["scale"]; ok {
+		scale, _ = strconv.Atoi(scaleStr)
+	}
+
+	rancherLabels := map[string]interface{}{}
+
+	for k, v := range lbConfig.Annotations {
+		if strings.HasPrefix(k, rancherSchedulerPrefix) {
+			key := rancherSchedulerPrefix + strings.Replace(k[len(rancherSchedulerPrefix):], ".", ":", 1)
```

Comment:
@wlan0 can we extract this prefix concatenation + replacement to its own method? as the same logic is being called multiple times